tests/integration/test_arrowflight_interface/test.py

```diff
@@ -0,0 +1,357 @@
+# coding: utf-8
+
+import os
+import pytest
+import pyarrow as pa
+import pyarrow.flight as flight
+
+from helpers.cluster import ClickHouseCluster, get_docker_compose_path
+from helpers.test_tools import TSV
+
+SCRIPT_DIR = os.path.dirname(os.path.realpath(__file__))
+DOCKER_COMPOSE_PATH = get_docker_compose_path()
+
+
+cluster = ClickHouseCluster(__file__)
+node = cluster.add_instance(
+    "node",
+    main_configs=[
+        "configs/config.xml",
+        "configs/grpc_config.xml",
+        "configs/key.pem",
+        "configs/cert.pem",
+    ],
+    user_configs=["configs/users.xml"],
+)
+
+def get_client():
+    client = flight.FlightClient(f"grpc+tls://{node.ip_address}:8888", disable_server_verification=True)
```

Comment:
Can you explain why we disable server verification here?

Reply:
server may not accept self-signed certificates on the test (host does not match - IP address is automatically generated)